cluster-state-service/Makefile

```diff
@@ -19,10 +19,10 @@ ROOT := $(shell pwd)
 .PHONY: all
 all: clean generate build unit-tests
 
-.PHONY: generate-models
-generate-models:
-	./scripts/v1/generate_swagger_models.sh
-	@echo "Generated swagger models"
+.PHONY: generate-swagger-artifacts
+generate-swagger-artifacts:
```

Comment:
Should we make it to regenerate swagger artifacts when swagger.json has changed like we did for the DS? https://github.com/blox/blox/blob/dev/daemon-scheduler/Makefile#L37